lbry/tests/integration/test_other_commands.py

```diff
@@ -10,3 +10,25 @@ class AddressManagement(CommandTestCase):
         single = await self.out(self.daemon.jsonrpc_address_list(addresses['items'][11]['address']))
         self.assertItemCount(single, 1)
         self.assertEqual(single['items'][0], addresses['items'][11])
+
+class SettingsManagement(CommandTestCase):
+
+    async def test_settings(self):
+        settings = self.daemon.jsonrpc_settings_get()
+        self.assertNotEqual(settings['lbryum_servers'][0],('127.0.0.1', 50002))
+        self.assertEqual(settings['lbryum_servers'][0],('127.0.0.1', 50001))
+
+        servers = ['server:50001', 'server2:50001']
+        setting = self.daemon.jsonrpc_settings_set('lbryum_servers', servers)
+        self.assertEqual(setting['lbryum_servers'][0], ('server', 50001))
+
+        self.daemon.jsonrpc_settings_set('use_upnp', True)
+        settings2 = self.daemon.jsonrpc_settings_get()
+        self.assertEqual(settings2['lbryum_servers'][0], ('server', 50001))
+        self.assertEqual(settings2['use_upnp'], True)
+
+        clearing = self.daemon.jsonrpc_settings_clear('lbryum_servers')
+        self.assertEqual(clearing['lbryum_servers'][0], ('spv1.lbry.com', 50001))
+        settings3 = self.daemon.jsonrpc_settings_get()
+        self.assertNotEqual(settings3['lbryum_servers'][0], ('spv1.lbry.com', 50002))
+        self.assertEqual(settings3['lbryum_servers'][0], ('spv1.lbry.com', 50001))
```

Comment:
There is a bit too much redundancy here, I think you can significantly simplify this and get the same logical coverage with:
```
        default = ('127.0.0.1', 50001)
        self.assertEqual(self.daemon.jsonrpc_settings_get()['lbryum_servers'][0], default)

        setting = self.daemon.jsonrpc_settings_set('lbryum_servers', ['server:50001'])
        self.assertEqual(setting['lbryum_servers'][0], ('server', 50001))

        setting = self.daemon.jsonrpc_settings_clear('lbryum_servers')
        self.assertEqual(setting['lbryum_servers'][0], default)
```

Reply:
Alright.
This brings up a question about what settings_clear should mean.
The above example expects that settings_clear return the lbryum_servers to "default = ('127.0.0.1', 50001) but it actually returns it to the conf coded values [('spv1.lbry.com, 50001), ...].

I think it's fine that clear always return it to the default conf values. Anyone who knows what they're doing will set it to what they need.